pandas/tests/frame/test_arithmetic.py

```diff
@@ -1577,3 +1577,17 @@ def test_arith_reindex_with_duplicates():
     result = df1 + df2
     expected = pd.DataFrame([[np.nan, 0, 0]], columns=["first", "second", "second"])
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("arg1", [pd.DataFrame({"x": [1, 2], "y": [1, 2]})])
```

Comment:
if there's just one of these (and i think there only needs to be one), then just put it in the test function instead of parameter

Reply:
Thanks for spotting this! Moved the DataFrame inside the test.